### DISCUSSION TITLE
Is it possible i want border rounded properties hover

### DISCUSSION THREAD

**Discussion by madfortech** [twbs/bootstrap#39875] [Q&A]
`<a class="link-offset-2 link-offset-3-hover link-underline link-underline-opacity-0 link-underline-opacity-75-hover" href="#">
  Underline opacity 0
</a>`
If i i want border rounded properties hover without custom css how is it possible ?

**Comment by julien-deramond** [comment]
Hi @madfortech 
There aren't any utilities/helpers like this one by default in the framework. For that, you can build your own with our [Utility API](https://getbootstrap.com/docs/5.3/utilities/api/) if you have a Sass build. Otherwise, when loading Bootstrap with a CDN (or other), you'll have to build your own classes our custom CSS.
